src/core/opamConsole.ml

```diff
@@ -595,8 +595,8 @@ let header_width () = min 80 (OpamStd.Sys.terminal_columns ())
 
 let header_msg fmt =
   let utf8camel = "\xF0\x9F\x90\xAB " in (* UTF-8 <U+1F42B, U+0020> *)
-  let padding = "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-\
-                 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=" in
+  let padding = "-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_\
+                 -_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-" in
```

Comment:
Out of curiosity, why?

Reply:
Just refreshing the header style once in a while. There is also a practical trick: it allows me to have an idea of the opam versions involved in any scrap of logs without proper `config report`... often helped, actually ;)